examples/next/sandbox.config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "infiniteLoopProtection": true,
+  "hardReloadOnChange": false,
+  "view": "browser",
```

Comment:
Is this necessary? These are the [defaults](https://codesandbox.io/docs/configuration#sandbox-configuration).

Reply:
Ah, this is not then, I simply copied the generated config from codesandbox. I'll remove the unnecessary keys, thanks!